main.nf

```diff
@@ -418,4 +515,68 @@ process stringtieFPKM {
 }
 
 
+/*
+ * STEP 10 MultiQC
+ */
+
+process multiqc { 
+    module 'bioinfo-tools'
+    module 'MultiQC/0.5'
```

Comment:
Better to avoid specifying versions here I think. Doesn't really matter for MultiQC, so better to just pull in the latest version.